tests/conftest.py

```diff
@@ -25,6 +25,7 @@
     from ape.types import AddressType
 
 
+# TODO: Refactor to using Ape's built-in temporary data folder feature
 DATA_FOLDER = Path(mkdtemp()).resolve()
```

Comment:
Ya, otherwise the temp path isnt cleaned up, tho i think the Os must delete them eventually